guest-owner-proxy/pre-attestation.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+message BundleRequest {
+	bytes PlatformPublicKey = 1;
+	bytes CertificateChain = 2;
+	int32 Policy = 3;
+}
+
+message BundleResponse {
+	bytes GuestOwnerPublicKey = 1;
+	bytes LaunchBlob = 2;
+	int32 ConnectionId = 3;
+}
+
+message SecretRequest {
```

Comment:
those are needed to calculate the firmware digest, which the GOP currently does not compute. the guest owner will need to know the kernel params when they calculate allowed digests and upload them to the GOP. hopefully they aren't too dynamic. if that becomes a problem we could tweak things so that the guest owner uploads the ovmf measurement, kernel measurement, and initrd measurements separately and then the GOP adds in the kernel params measurement and calculates the final measurement. we would need to come up with some way for the guest owner to provide a policy to validate certain kernel params.